### ISSUE TITLE
emoji in label 

### ISSUE THREAD

**Issue by miduch** [usefulness/easylauncher-gradle-plugin#409]
Is it possible to use emoji in label ? 

I tried it like this : 

filters = chromeLike("Abc ⚡")
filters = chromeLike("Abc \uD83D\uDD25")

but no success, end result is like this : 
<img width="16" alt="Screenshot 2023-01-20 at 16 00 09" src="https://user-images.githubusercontent.com/22243296/213715320-b9d5cb99-aba7-4989-8229-5ed57df62eea.png">

**Comment by mateuszkwiecinski** [comment]
It should be possible, but you would have to use a font that supports that emoji 👀  by default the pluin uses Roboto, which doesn't seem to support it: https://fonts.google.com/specimen/Roboto?preview.text=%F0%9F%99%88%F0%9F%91%8F%E2%9D%A4%EF%B8%8F%E2%9C%85%E2%9A%A1&preview.text_type=custom

**Comment by mateuszkwiecinski** [comment]
Aaaand I see already I was wrong 😛 It doesn't seem like the font is responsible for "drawing" emojis. I'll have to dig a bit further 👀

**Comment by mateuszkwiecinski** [comment]
Ooor maybe I wasn't that way off? I played with different fonts and I somewhat made it work:
```groovy
filters chromeLike(label: "❤️🙈👏✅⚡️", font: "Apple Color Emoji"),
 ```
 but for some reason, they don't behave in an expected way, and I don't really understand why. 
 The `Apple Color Emoji`, which I believe is the default font my system used for displaying emojis produces following output:
 
<img width="222" alt="image" src="https://user-images.githubusercontent.com/36954793/213858301-76ddf9b0-9764-4939-a62b-f34142f48183.png">

so then I tried to switch to [Noto Color Emoji](https://fonts.google.com/noto/specimen/Noto+Color+Emoji?preview.text=%F0%9F%99%88%F0%9F%91%8F%E2%9D%A4%EF%B8%8F%E2%9C%85%E2%9A%A1&preview.text_type=custom&query=emoji), but it doesn't show any characters for me :/ 

I also accidentally used an invalid font name, which produced even stranger output, with heart not being rendered in color and system falling back to font called `Dialog`
<img width="247" alt="image" src="https://user-images.githubusercontent.com/36954793/213858436-f8bd93f3-2067-4150-b2cc-25cbf7e001c4.png">

To summarize, I don't understand how emojis work, and the more I read the dumber I get 😛 As a temporary workaround I can suggest playing with different fonts, and some of them should work in your case 🤷 For example this:
`chromeLike(label: "Abc ⚡", font: "foo font, doesn't exist")` produces following output for me locally
<img width="210" alt="image" src="https://user-images.githubusercontent.com/36954793/213858505-7d0d2c01-1f72-40bc-8a58-e8dc5fd4b3da.png">

**Comment by miduch** [comment]
Let's wait for some kind heart emoji expert 😜
